solutions/fully-configurable/catalogValidationValues.json.template

```diff
@@ -0,0 +1,8 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "existing_kms_instance_crn": $HPCS_US_SOUTH_CRN,
+  "region": "us-south",
+  "prefix": $PREFIX,
+  "existing_cos_instance_crn": $COS_INSTANCE_CRN,
+  "existing_resource_group_name": "geretain-test-resources"
```

Comment:
I would expect the above config to fail validation because you have not set the encryption boolean to true